metrics/grafana/pd.json

```diff
@@ -2692,7 +2692,7 @@
           },
           "yaxes": [
             {
-              "format": "opm",
+              "format": "ops",
```

Comment:
Is it possible to use opm since the store limit is based on minutes?

Reply:
I think it's both OK, but the units need to be the same for at least one plate.